chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsReceivedIntegrationTest.kt

```diff
@@ -41,17 +43,18 @@ internal object JoinedRoomsReceivedIntegrationTest : Spek({
 
     val simpleUser = UserApiType(
         id = "user1",
-        createdAt = "",
-        updatedAt = "",
+        createdAt = "2020-03-12T14:33:20Z",
+        updatedAt = "2020-03-12T14:33:20Z",
         name = "name",
-        avatarURL = null,
+        avatarUrl = null,
         customData = null,
         online = false
     )
 
     describe("given existing state with one room") {
         val dateApiTypeMapper = DateApiTypeMapper()
         val joinedRoomApiTypeMapper = JoinedRoomApiTypeMapper(dateApiTypeMapper)
+        val userApiTypeMapper = UserApiTypeMapper(dateApiTypeMapper)
```

Comment:
likewise one line up would be better

Reply:
👍 have updated in both occurrences, and updated both the constructors